client/src/Components/Common/Navbar/index.js

```diff
@@ -43,9 +41,17 @@ function Navbar() {
     setAnchorEl(null);
   };
 
+  const handleLogout = () => {
+    logout((err) => {
+      if (!err) {
+        history.push('/');
```

Comment:
I guess no need to do that as we're setUSer to null, So our router will direct user if not allowed to be in that page